packages/example-app-base/src/components/ExampleCard/ExampleCard.types.ts

```diff
@@ -27,6 +27,12 @@ export interface IExampleCardProps {
 
   /** Optional override styles */
   styles?: IStyleFunctionOrObject<IExampleCardStyleProps, IExampleCardStyles>;
+
+  /** On click handler to ensure only one code editor instance is shown at once */
+  onClick?: (card: string) => void;
```

Comment:
This should have a more specific name, like `onShowHideCodeClick`. And maybe an additional boolean parameter to indicate if it's being shown or hidden.

Reply:
Or instead of adding a boolean parameter, you could add documentation indicating that if the code is being hidden, pass undefined/an empty string.

Reply:
@ecraig12345 Was so caught up in switching between different editors that I didn't even realize basic toggle hide didn't work, pushed fix :)